### ISSUE TITLE
Problem with PHPStan

### ISSUE THREAD

**Issue by TristanPouliquen** [api-platform/api-platform#2101]
**API Platform version(s) affected**: 2.6.7

**Description**  
After updating dependencies to last version, we get these errors in our PHPStan installation:
```
ParseError (syntax error, unexpected 'static' (T_STATIC)) thrown        
         while looking for class ApiPlatform\Core\Util\ClientTrait80.            
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols   
```

The PHPStan message isn't very clear, so I'm not sure of what causes the issue as the code inside `ClientTrait80` seems fine.

Or is it the `static` return type on the `withOptions` method that raises such an alarm?

We are running under PHP7.4

**Comment by dpinheiro** [comment]
Having the same problem

**Comment by dpinheiro** [comment]
btw, just found out this is a duplicate of https://github.com/api-platform/core/issues/4614, and an issue has been already created on phpstan repo https://github.com/phpstan/phpstan/issues/6386

guess this could be closed

**Comment by TristanPouliquen** [comment]
Well spotted, I hadn't found it! :-) Closing the issue then

**Comment by ondrejmirtes** [comment]
This is going to be fixed in PHPStan 1.4.2 which is going to be released soon.

**Comment by ondrejmirtes** [comment]
Released: https://github.com/phpstan/phpstan/releases/tag/1.4.2
